mhubert-pmn/Back-Url-Shortener
# Devops/dockerize node

## Task
Correction Dockerize Back

## Type of change

## Description
- Corrected docker-compose files
- Modified directory for datadir
- Modified dump name

## Testing scenario
In the terminal (you have to be in the back-url-shortener folder):
- Use "docker-compose down"
- Use "docker-compose up --build" to create new images
- see database at http://localhost:8301/ with phpmyadmin
- Down and delete running containers, volumes and networks: `docker-compose down`

## Trello card
[https://trello.com/c/0WYAxxfO](https://trello.com/c/0WYAxxfO)
